db/schema.rb

```diff
@@ -39,6 +39,16 @@
     t.index ["singleton_guard"], name: "index_myconfigs_on_singleton_guard", unique: true
   end
 
+  create_table "offsprings", force: :cascade do |t|
+    t.string   "first_name", null: false
+    t.string   "last_name", null: false
+    t.string   "type", null: false
```

Comment:
I think you need to be either more specific with which line you want with not null or be coherent with the previous comments you made. This is part of the corrections you wrote to @agarciavera  in a previous PR (aready closed #41)
![image](https://cloud.githubusercontent.com/assets/19265840/18912017/83055c00-8580-11e6-8d35-577028777125.png)